site/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "swag opportunities for developers",
   "main": "index.js",
   "scripts": {
+    "start": "gulp",
```

Comment:
Please remove this NPM script, #76 gonna add it :wink:

Reply:
Done in e9988539953c94a6a4e90283c48f670014c1d9ef 👌🏻